src/data/navigation/header.js

```diff
@@ -9,10 +9,29 @@ module.exports = [
     },
     {
       title: "REST",
-      path: "/rest/",
+      menu: [
+        {
+          title: "Guide",
+          path: "/rest/",
+        },
+        {
+          title: "Reference",
+          path: "/rest/quick-reference/", EventTarget: "_blank"
```

Comment:
I think you can remove the `EventTarget:` prop for REST. It's not opening in a new tab like it does for GraphQL. It could be because the target is in the left nav, unblike GraphQL. Regardless, @jhadobe will likely revisit when he adds the ACCS schemas and optimizes the PaaS schemas for full width rednering.